exchanges/kucoin/kucoin_test.go

```diff
@@ -1973,29 +1974,15 @@ func TestPushData(t *testing.T) {
 	testexch.FixtureToDataHandler(t, "testdata/wsHandleData.json", ku.wsHandleData)
 }
 
-func verifySubs(tb testing.TB, subs subscription.List, a asset.Item, prefix string, expected ...string) {
-	tb.Helper()
-	var sub *subscription.Subscription
-	for i, s := range subs {
-		if s.Asset == a && strings.HasPrefix(s.Channel, prefix) {
-			if len(expected) == 1 && !strings.Contains(s.Channel, expected[0]) {
-				continue
-			}
-			if sub != nil {
-				assert.Failf(tb, "Too many subs with prefix", "Asset %s; Prefix %s", a.String(), prefix)
-				return
-			}
-			sub = subs[i]
-		}
-	}
-	if assert.NotNil(tb, sub, "Should find a sub for asset %s with prefix %s for %s", a.String(), prefix, strings.Join(expected, ", ")) {
-		suffix := strings.TrimPrefix(sub.Channel, prefix)
-		if len(expected) == 0 {
-			assert.Empty(tb, suffix, "Sub for asset %s with prefix %s should have no symbol suffix", a.String(), prefix)
-		} else {
-			currs := strings.Split(suffix, ",")
-			assert.ElementsMatch(tb, currs, expected, "Currencies should match in sub for asset %s with prefix %s", a.String(), prefix)
-		}
+var subPairs currency.Pairs
+
+func init() {
```

Comment:
1. I use multiple init functions next to the thing they're acting upon because action close to usage is clearer. I think it's one of the go tenents, in fact. So have as many init() funcs as you have things that you want to init next to where they are used.

2. No reason to do this at all. It's reducible complexity; At one point subPairs was used elsewhere.
Thanks for spotting 🙏 

Fixed bc20106a